integration_test/ops_agent_test.go

```diff
@@ -465,74 +465,95 @@ func TestHTTPRequestLog(t *testing.T) {
 			"status":        200,
 		}
 
-		// Write a log with an http request in the message to cloud logging and immediately query it.
-		writeAndQueryLog := func(httpRequestKey string, logId string) *cloudlogging.Entry {
-			logBody := map[string]interface{}{
-				"logId":        logId,
-				httpRequestKey: httpRequestBody,
-			}
-			logBytes, err := json.Marshal(logBody)
-			if err != nil {
-				t.Fatalf("could not marshal test log: %v", err)
-			}
-			err = gce.UploadContent(
-				ctx,
-				logger,
-				vm,
-				strings.NewReader(string(logBytes)+"\n"),
-				logPath)
-			if err != nil {
-				t.Fatalf("error writing log line: %v", err)
-			}
+		// Log with HTTP request data nested under "logging.googleapis.com/httpRequest".
+		// const newHTTPRequestKey = confgenerator.HttpRequestKey
+		const newHTTPRequestKey = "logging.googleapis.com/http_request"
```

Comment:
Should this be the opposite? aka:

`newHTTPRequestKey` => `logging.googleapis.com/HttpRequest` or `confgenerator.HttpRequestKey`
`oldHTTPRequestKey` => `logging.googleapis.com/http_request`

Reply:
Fixed in [59b0a82](https://github.com/GoogleCloudPlatform/ops-agent/pull/655/commits/59b0a8260d13e87b6f495b888cd28d5f2e8866c3)